tests/density_estimator_test.py

```diff
@@ -280,7 +280,7 @@ def test_correctness_of_density_estimator_log_prob(
         input_sample_dim,
     )
     log_probs = density_estimator.log_prob(inputs, condition=condition)
-    assert torch.allclose(log_probs[0, :], log_probs[1, :])
+    assert torch.allclose(log_probs[0, :], log_probs[1, :], rtol=1e-4)
```

Comment:
Why? (I guess the new NFs, which use numerical inversion or whatever it might get relevant)

Reply:
Locally, the log probs looked identical up to the 4 digit. I thought rtol would be better than atol because log probs can get quite small? 
`rtol=1e-4` allows errors up to 0.0001 % no?

Reply:
it was failing for me locally. Changing rtol from the defautl 1e-5 to 1e-4 fixed it.